luminaire/exploration/data_exploration.py

```diff
@@ -1,3 +1,5 @@
+from luminaire.model.model_utils import LADHolidays
```

Comment:
Later we might want to think to rename LADHolidays to LADUSHolidays. Or how we users can pass region specific holidays.

Reply:
This is a good suggestion. I'll create an issue